src/ol/renderer/dom/domvectorlayerrenderer.js

```diff
@@ -93,6 +93,16 @@ goog.inherits(ol.renderer.dom.VectorLayer, ol.renderer.dom.Layer);
 /**
  * @inheritDoc
  */
+ol.renderer.dom.VectorLayer.prototype.clearFrame = function() {
+  this.context_.canvas.width = 0;
+  this.context_.canvas.height = 0;
```

Comment:
It used to be that the `width` (or `height`) setting trick was consistently faster.  It looks like this is not consistent any more (http://jsperf.com/canvasclear/30).

I don't think it is worth changing, but I think the following would be more straightforward:
```js
var canvas = this.context_.canvas;
this.context_.clearRect(0, 0, canvas.width, canvas.height);
```

This would avoid any (potential future?) issues with setting height/width to 0.  If we do want to use the `width` (or `height`) trick, it would be enough to do this:

```js
var canvas = this.context_.canvas;
canvas.width = canvas.width;
```